<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1260</width>
      <height>670</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Resources/Images/Background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="MIKATA">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>410</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Resources/Images/MIKATA.png</pixmap>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>120</y>
      <width>350</width>
      <height>440</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
border : 2px solid rgb(29, 186, 255);
border-radius : 10px;
background-color: rgba(0, 0, 0, 128);
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPlainTextEdit" name="chatbox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>350</width>
       <height>390</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BankGothic Lt BT</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 0px;
color: rgb(255, 255, 255);
background-color: rgba(0,0,0,10);
QScrollBar: {
 border : 2px solid red; background : black; width :15px ;
 }
QScrollBar::handle:vertical {
 background : black; min-height : 30px; 
}</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="command">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>397</y>
       <width>340</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>BankGothic Lt BT</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(0, 0, 0, 64);
border : 2px solid rgb(0, 174, 218);
border-radius : 10px;
color : white;
padding-right: 30px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="snd_button">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>400</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Resources/Images/Send Button.png</normaloff>Resources/Images/Send Button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="prev_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Images/Previous Button.png</normaloff>Resources/Images/Previous Button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="play_button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>500</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Images/Platy Button.png</normaloff>Resources/Images/Platy Button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="next_button">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>500</y>
      <width>80</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Images/Next Button.png</normaloff>Resources/Images/Next Button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>70</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pause_button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>500</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>Resources/Images/Pause Button.png</normaloff>Resources/Images/Pause Button.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
